pages/docs/getting-started.mdx

```diff
@@ -1,6 +1,6 @@
 ---
 title: "Getting Started"
-date: "2021-02-08"
```

Comment:
I forgot to update this, but this page contains the same video and it was updated in #873